### PR TITLE
sdn: add metric for "failed to find netid for namespace: XXXX in vnid map" errors

### PR DESCRIPTION
**Pull request by dcbw**
@eparis @openshift/networking

### CODE REVIEW

**Comment by knobunc** [comment]
This is to allow us to better monitor https://bugzilla.redhat.com/show_bug.cgi?id=1509799

**Comment by knobunc** [comment]
@openshift/networking FYI

File: pkg/network/node/metrics.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ const (
 	PodSetupLatencyKey          = "pod_setup_latency"
 	PodTeardownErrorsKey        = "pod_teardown_errors"
 	PodTeardownLatencyKey       = "pod_teardown_latency"
+	VnidNotFoundErrorsKey       = "vnid_not_found_errors"
```
Comment (danwinship) [inline pkg/network/node/metrics.go]: `VNIDNotFoundErrorsKey`

Comment (eparis) [reply pkg/network/node/metrics.go]: can we fix in a different patch so we don't have to restart CI?

**Pull request merged**